cmd/scollector/doc.go

```diff
@@ -24,6 +24,9 @@ The flags are:
 	-f=""
 		Filters collectors matching these terms, separated by
 		comma. Overrides Filter in conf file.
+	-F=""
```

Comment:
I don't have any specific reason to keep the flag, it was just easier for me to test it this way. I'll delete it